### DISCUSSION TITLE
Can't sign-in to my supabase account

### DISCUSSION THREAD

**Discussion by Mohamedzh** [supabase/supabase#9463] [Questions] [answered]
I can't login to my supabase account. Whenever I click "sign in" or "sign in with github", i get redirected to the same page with a hash '#' at the end of the url.

**Comment by Sukikiroi** [comment, marked as answer]
Hello,
Try Clear the Browser Data of  supabase page

**Reply by Mohamedzh** [reply]
Thanks
It solved my sign-in issue
